transition-scripts/test-dns.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+usage() {
+    cat <<EOF
+Test that a url points at a specific DNS
+
+Usages:
+    $0 url cluster
```

Comment:
do we really need cluster input? we can just check the dns and output the cluster name